code/modules/mob/living/simple_animal/hostile/mining_mobs/elites/legionnaire.dm

```diff
@@ -48,6 +48,7 @@
 	var/mob/living/simple_animal/hostile/asteroid/elite/legionnairehead/myhead = null
 	var/obj/structure/legionnaire_bonfire/mypile = null
 	var/has_head = TRUE
+	var/charging
```

Comment:
[add dmdoc to meet documentation requirements](https://github.com/tgstation/tgstation/blob/master/.github/AUTODOC_GUIDE.md)